test/e2e/plain_provisioner_test.go

```diff
@@ -882,9 +882,33 @@ var _ = Describe("plain provisioner bundleinstance", func() {
 
 			err = c.Create(ctx, biOriginal)
 			Expect(err).To(BeNil(), "failed to create original bundle instance")
-			Eventually(func() error { // Guarantee that the original bundle instance is create prior to the duplicate
-				return c.Get(ctx, client.ObjectKeyFromObject(biOriginal), &rukpakv1alpha1.BundleInstance{})
-			}).Should(Succeed(), "failed to validate original bundle instance was created in time")
+
+			// ensure the original BI owns the underlying bundle before creating the duplicate
+			By("projecting a successful installation status for the original BundleInstance")
+			Eventually(func() bool {
+				if err := c.Get(ctx, client.ObjectKeyFromObject(biOriginal), biOriginal); err != nil {
+					return false
+				}
+				if biOriginal.Status.InstalledBundleName != "" {
```

Comment:
Is this mistake?  Line 907 is also checking `biOriginal.Status.InstalledBundleName`.

Reply:
Yup that's a good catch -- I consolidated it to check only once.